radix-common/src/data/manifest/model/manifest_expression.rs

```diff
@@ -20,25 +20,72 @@ pub enum ManifestExpression {
 
 #[cfg_attr(feature = "fuzzing", derive(Arbitrary))]
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
-pub enum BucketBatch {
+pub enum ManifestBucketBatch {
     ManifestBuckets(Vec<ManifestBucket>),
     EntireWorktop,
 }
 
-impl BucketBatch {
+pub trait ResolvableBucketBatch {
```

Comment:
Let's rename to `ResolvableManifestBucketBatch` for consistency with the other types.